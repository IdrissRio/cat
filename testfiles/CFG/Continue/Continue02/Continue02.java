public class Continue02 {
  void m() {
    for (int f;;) {
      int c;
      continue;
    }
  }
}