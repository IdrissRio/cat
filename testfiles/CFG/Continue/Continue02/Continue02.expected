NodeName: Entry[15] Successors: [Exit[16]]
NodeName: Exit[16] Successors: []
NodeName: Entry[7] Successors: [Exit[8]]
NodeName: Exit[8] Successors: []
NodeName: Entry[29] Successors: [VariableDeclarator[45]]
NodeName: VariableDeclarator[45] Successors: [ImplicitCondition[63]]
NodeName: ImplicitCondition[63] Successors: [VariableDeclarator[57], Exit[36]]
NodeName: VariableDeclarator[57] Successors: [ImplicitCondition[63]]
NodeName: Exit[36] Successors: []

NodeName: Exit[16] Predecessor: [Entry[15]]
NodeName: Entry[15] Predecessor: []
NodeName: Exit[8] Predecessor: [Entry[7]]
NodeName: Entry[7] Predecessor: []
NodeName: Exit[36] Predecessor: [ImplicitCondition[63]]
NodeName: ImplicitCondition[63] Predecessor: [VariableDeclarator[45], VariableDeclarator[57]]
NodeName: VariableDeclarator[45] Predecessor: [Entry[29]]
NodeName: Entry[29] Predecessor: []
NodeName: VariableDeclarator[57] Predecessor: [ImplicitCondition[63]]
